egs/libri_2_mix/ConvTasNet/run.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e  # Exit on error
+# Main storage directory. You'll need disk space to dump the raw LibriSpeech flac files
+# and generate the sources and mixtures
+storage_dir=/srv/storage/talc@talc-data.nancy/multispeech/calcul/users/jcosentino/DATA/
+
+# If you haven't downloaded and extracted the LibriSpeech dataset start from stage 1
+
+# If you already have dowloaded and extracted LibriSpeech,
+# specify the path to the directory here and start from stage 2
+LibriSpeech_dir=${storage_dir}
```

Comment:
Create a Librispeech directory inside storage_dir (at stage 1), do all the downloads there. LibriSpeech_dir would default to ${storage_dir}/LibriSpeech. 
Also, no caps in variables please : LibriSpeech_dir --> librispeech_dir